src/main/java/lab3/ui/Console.java

```diff
@@ -0,0 +1,31 @@
+package lab3.ui;
+import java.util.Scanner;
+import lab3.game.Game;
+public class Console {
+
+    // TODO: create a CLASS method to display a tictactoe board
+
+    // TODO: create a CLASS method to get the next move from a player
+    private Game game;
+    private Scanner scanner;
+
+    public Console() {
+        this.game = new Game();
+        this.scanner = new Scanner(System.in);
+    }
+
+    public void startGame() {
+        while (!game.isGameOver()) {
+            game.printBoard();
+            System.out.println("Enter row (0-2) and column (0-2) separated by space: ");
+            int row = scanner.nextInt();
+            int col = scanner.nextInt();
```

Comment:
`nextInt` throws an exception if the user doesn't enter a valid int, and you are not handling this with a `try/catch`